tests/unit/test_xml.py

```diff
@@ -289,7 +293,1401 @@
     } ]
 PIPELINE_CONFIG = 'remote hop server'
 PIPELINE = 'pipelines/fake-data-generate-person-record.hpl'
+PIPELINE_XML = '''<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should I use a .xml file or is it okay this way?

Reply:
It's OK.